types/plur/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for plur 3.0
+// Project: https://github.com/sindresorhus/plur#readme
+// Definitions by: Kyle Roach <https://github.com/iRoachie>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export default function plur(
```

Comment:
The correct way to export this, based on the implementation, would be

```ts
function plur( 
    word: string,
    plural: string | number,
    count?: number ): string
export = plur;
```

Reply:
That fails from tsc. A 'declare' modifier is required for a top level declaration in a .d.ts file.

Reply:
<img width="756" alt="screen shot 2018-09-07 at 8 07 49 pm" src="https://user-images.githubusercontent.com/5962998/45247958-b9d21380-b2d9-11e8-8df2-c57f15cc900d.png">

Reply:
Yes, youll want to add `declare` in the beginning.